test/tests/misc/save_in/tests

```diff
@@ -10,12 +10,12 @@
   [./test_soln_var_err]
     type = RunException
     input = 'save_in_soln_var_err_test.i'
-    expect_err = 'Trying to use solution variable u as a save_in variable in diff'
+    expect_err = 'diff/save_in\) cannot use solution variable as save-in variable'
   [../]
   [./test_diag_sol_var_err]
     type = RunException
     input = 'diag_save_in_soln_var_err_test.i'
-    expect_err = 'Trying to use solution variable u as a diag_save_in variable in diff'
+    expect_err = 'diff/diag_save_in\) cannot use solution variable as diag save-in variable'
```

Comment:
This is OK, but it's like a scratch you can't itch (unbalanced parenthesis)! 
Can you change the match to \(.*diff/diag_save_in\)? Actually we should probably decide if this is the format we want (input syntax showing up in parenthesis). Maybe we should use brackets or instead since that's the symbol in the input file? In either case, I'd prefer to see more matched here.

Reply:
Brackets would be a bit weird because the actual parameter names are not part of the block/section names.  The actual hit equivalent syntax for the parameter is without brackets or parens (i.e. ``foo/bar=baz``).